lib/tasks/email_domain_actions.rake

```diff
@@ -0,0 +1,144 @@
+# -*- encoding : utf-8 -*-
+namespace :email_domain_actions do
+
+  desc "Lists top 20 most used email domains"
+  task :top20 => :environment do
+    # the limit of 20 is fairly arbitrary but I imagine this being more
+    # annoying than useful otherwise
+
+    from = ENV["START_DATE"]
+
+    sql = if from
```

Comment:
calling the lib class `UserStats` for now - happy to rename it later but just `Users` felt too close to  key model name